trinity/utils/datastructures.py

```diff
@@ -512,24 +513,30 @@ def register_tasks(self, tasks: Tuple[TTask, ...]) -> None:
         initialize this intance with: ``accept_dangling_tasks=True``.
 
         :param tasks: the tasks to register, in iteration order
+        :param ignore_duplicates: any tasks that have already been registered will be ignored,
+            whether ready or not
```

Comment:
is the indentation intentional?

Reply:
Yeah, not sure if it's required, but it is easier for me to read like:
```
:param name: some description here that is too long for one line, plus
    a few trailing words from the previous description
:param name2: more description